### PR TITLE
Implemented code to display OMS information on settings page(#1ytrtk0)

### PR DESCRIPTION
**Pull request by disha1202**
### Related Issues

Closes #

### Screenshots of Visual Changes before/after (If There Are Any)

**IMPORTANT NOTICE** - Remember to update `CHANGELOG.md` with description of your change

### Contribution and Currently Important Rules Acceptance

- [ ] I read and followed [contribution rules](https://github.com/DivanteLtd/vsf-capybara/blob/master/CONTRIBUTING.md)
- [ ] I have changed the target to hacktoberfest branch

### CODE REVIEW

File: src/views/Settings.vue [hunk 1]
```diff
@@ -17,6 +17,13 @@
           </ion-select>
         </ion-item>
 
+        <!-- OMS information -->
+      <ion-item>
+        <ion-icon :icon="codeWorkingOutline" slot="start"/>
+        <ion-label>{{ $t("OMS") }}</ion-label>
+        <ion-note slot="end">{{instanceUrl}}</ion-note>
```
Comment (adityasharma7) [inline src/views/Settings.vue:24]:
```suggestion
        <ion-note slot="end">{{ instanceUrl }}</ion-note>
```

File: src/views/Settings.vue [hunk 2]
```diff
@@ -17,6 +17,13 @@
           </ion-select>
         </ion-item>
 
+        <!-- OMS information -->
+      <ion-item>
```
Comment (adityasharma7) [inline src/views/Settings.vue:21]: It seems there are indentation issues

Comment (disha1202) [reply src/views/Settings.vue]: Fixed indentation issues

File: src/views/Settings.vue [hunk 3]
```diff
@@ -17,6 +17,13 @@
           </ion-select>
         </ion-item>
 
+        <!-- OMS information -->
+        <ion-item>
+          <ion-icon :icon="codeWorkingOutline" slot="start"/>
+          <ion-label>{{ $t("OMS") }}</ion-label>
+          <ion-note slot="end">{{instanceUrl}}</ion-note>
```
Comment (adityasharma7) [inline src/views/Settings.vue:24]: Fix spacing

**Review by adityasharma7 — CHANGES_REQUESTED** [review]

File: src/views/Settings.vue (hunk 3 above)
Comment (disha1202) [reply src/views/Settings.vue]: Fixed spacing

**Review by adityasharma7 — APPROVED** [review]

**Pull request merged**